File: Betch-2-1/Labs/DataStructursLab/Experment-1/LinearSearch.py
class LinearSearch:
  def __init__(self,l,k):
    self.arr=l
    self.key=k
  @staticmethod
  def _Searching(self):
    f=0
    for i in range(len(self.arr)):
      if self.key==self.arr[i]:
        f+=1
        break
    else:
      print("element is not found")
    if f:
      print(f"element is found at the location:{i+1}")
  def find(self):
    LinearSearch._Searching(self)

obj=LinearSearch(l=list(map(int,input("enter the input list:").split())),k=int(input("enter key element:")))
obj.find()

File: Betch-2-1/ClassWork/DataStructurs/Unit-1/Searching/LinearSearch/LineraSearch in java/LinearSearch.java
public class LinearSearch {
    public static void main(String[] args) {
        int a[]={10,20,40,50};
        int key=50,f=0,i;
        for(i=0;i<a.length;i++){
            if(key==a[i]){
                f++;
                break;
            }
        }
        if(f!=0) System.out.println("element found at location:"+(i+1));
        else System.out.println("element not found");
    }
}
